static/css/gridpage.css

```diff
@@ -17,12 +17,18 @@
 
 .launch-card {
     position: relative;
-    width: 33%;
-    padding: 0 30px 0 0;
-    margin-bottom: 50px;
+    margin-bottom: 30px;
     min-height: 152px;
 }
 
+@media (min-width: 768px){
+    .launch-card {
+        width: 33%;
```

Comment:
Hello @Arhell .
I compared the changes to the current version of the docs home page (minimized desktop browser).
Here is what looks different with the changes:
- I see a vertical scroll bar in the vendorStrip
- An empty flyout-button appears on the left top corner (normally invisible)
- Text sections now left aligned.

Reply:
@kbhawkey 
Hi, of course, because the wizard already has more recent changes, need to do a rebase with master

Reply:
Please look only at these blocks.
As you can see by the code, they only changed